demo/sticky/content/playground

```diff
@@ -0,0 +1,39 @@
+const FixedNode = (
+  <div style={{ width: '100%', height: '100px', backgroundColor: 'lightblue', border: '2px solid blue', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+    <span style={{ color: 'blue' }}>This is a FIXED to BOTTOM element, with FULL-WIDTH size.</span>
+  </div>
+)
+
+const StickyNode = (
+  <div style={{ backgroundColor: 'lightpink', border: '2px solid red', width: '200px' }}>
+    <span style={{ color: 'red' }}>This is a STICKY element, fixed to TOP.</span>
+  </div>
+)
+
+const emptyLines = (n) => (
+  <div>
+    {[...Array(n)].map((_, i) => (
+      <span key={'span_' + i}>...<br /></span>
+    ))}
+  </div>
+)
+
+return (
+  <div style={{ height: '2000px', width: '100%', backgroundColor: 'white'}}>
+    { emptyLines(10) }
+
+    <StickyContent
+      sticky
+      scrollableElement={document.body}
```

Comment:
Can you put an example with a node that is not body but another element ??
Just to check the behavior when it's not the body.

Reply:
For the moment, the implementation don't support listening to scroll in any custom element.
This `scrollableElementSelector` property refers only to the DOM element to inspect in order to calculate the total distance scrolled in page (by getting its scrollTop value).